packages/core/src/amazonqFeatureDev/userFacingText.ts

```diff
@@ -21,3 +21,9 @@ export const updateCode = 'Code has been updated. Would you like to work on anot
 export const sessionClosed = 'Your session is now closed.'
 export const newTaskChanges = 'What change would you like to make?'
 export const uploadCodeError = `Amazon Q is unable to upload workspace artifacts to S3 for feature development. For more information, see the [Amazon Q documentation](${manageAccessGuideURL}) or contact your network or organization administrator.`
+
+// Utils for logging and showing customer facing conversation id text
+export const conversationIdChat = (conversationId?: string) =>
+    conversationId ? `\n\nConversation ID: **${conversationId}**` : ''
+export const conversationIdLog = (featureName: string = '', conversationId: string) =>
```

Comment:
Is there ever a case where we would pass in featureName? Isn't there a constant that just defines featureName for us?

Reply:
Good point, we can avoid the featureName passing as it is just a constant, I can hardcode that.